### PR TITLE
Cloud extensions

### LINKED ISSUE #393: Extensions available on Citus Cloud
## Why are we implementing it? (sales eng)

Clearly call out what extensions are supported.

### Good locations for content in docs structure

Somewhere under the cloud section

Supported extensions are:

                                          List of installed extensions
        Name        | Version |   Schema   |                            Description
--------------------+---------+------------+--------------------------------------------------------------------
 btree_gin          | 1.0     | public     | support for indexing common datatypes in GIN
 btree_gist         | 1.2     | public     | support for indexing common datatypes in GiST
 citext             | 1.3     | public     | data type for case-insensitive character strings
 citus              | 6.2-4   | pg_catalog | Citus distributed database
 cube               | 1.2     | public     | data type for multidimensional cubes
 dblink             | 1.2     | public     | connect to other PostgreSQL databases from within a database
 earthdistance      | 1.1     | public     | calculate great-circle distances on the surface of the Earth
 fuzzystrmatch      | 1.1     | public     | determine similarities and distance between strings
 hll                | 1.0     | public     | type for storing hyperloglog data
 hstore             | 1.4     | public     | data type for storing sets of (key, value) pairs
 intarray           | 1.2     | public     | functions, operators, and index support for 1-D arrays of integers
 ltree              | 1.1     | public     | data type for hierarchical tree-like structures
 pg_buffercache     | 1.2     | public     | examine the shared buffer cache
 pg_freespacemap    | 1.1     | public     | examine the free space map (FSM)
 pg_prewarm         | 1.1     | public     | prewarm relation data
 pg_stat_statements | 1.4     | public     | track execution statistics of all SQL statements executed
 pg_trgm            | 1.3     | public     | text similarity measurement and index searching based on trigrams
 pgcrypto           | 1.3     | public     | cryptographic functions
 pgrowlocks         | 1.2     | public     | show row-level locking information
 pgstattuple        | 1.4     | public     | show tuple-level statistics
 plpgsql            | 1.0     | pg_catalog | PL/pgSQL procedural language
 session_analytics  | 1.0     | public     |
 shard_rebalancer   | 6.2     | public     |
 sslinfo            | 1.2     | public     | information about SSL certificates
 tablefunc          | 1.0     | public     | functions that manipulate whole tables, including crosstab
 unaccent           | 1.1     | public     | text search dictionary that removes accents
 uuid-ossp          | 1.1     | public     | generate universally unique identifiers (UUIDs)
 xml2               | 1.1     | public     | XPath querying and XSLT

### PR DESCRIPTION
**Pull request by begriffs**
Fixes #393

### CODE REVIEW

File: cloud/extensions.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
++--------------------+---------+------------+--------------------------------------------------------------------+
+|        Name        | Version |   Schema   |                            Description                             |
++====================+=========+============+====================================================================+
+| btree_gin          | 1.0     | public     | support for indexing common datatypes in GIN                       |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| btree_gist         | 1.2     | public     | support for indexing common datatypes in GiST                      |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| citext             | 1.3     | public     | data type for case-insensitive character strings                   |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| citus              | 7.0-15  | pg_catalog | Citus distributed database                                         |
```
Comment (begriffs) [inline cloud/extensions.rst]: We may want to remove the version column because it could be a maintenance burden in the docs.

File: cloud/extensions.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pgcrypto           | 1.3     | public     | cryptographic functions                                            |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pgrowlocks         | 1.2     | public     | show row-level locking information                                 |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| pgstattuple        | 1.4     | public     | show tuple-level statistics                                        |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| plpgsql            | 1.0     | pg_catalog | PL/pgSQL procedural language                                       |
++--------------------+---------+------------+--------------------------------------------------------------------+
+| session_analytics  | 1.0     | public     |                                                                    |
```
Comment (sumedhpathak) [inline cloud/extensions.rst]: Do these two (`session_analytics` and `shard_rebalancer`) not have a description? If not, could you open an issue in each of those repo's to add one? (Or better yet, open up a PR to do so)

File: index.rst [hunk 3]
```diff
@@ -121,8 +121,10 @@ Welcome to the documentation for Citus 7.0! Citus horizontally scales PostgreSQL
    cloud/availability.rst
    cloud/security.rst
    cloud/scaling.rst
+   cloud/forking.rst
```
Comment (sumedhpathak) [inline index.rst]: Why this change?

File: cloud/extensions.rst (hunk 2 above)
Comment (begriffs) [reply cloud/extensions.rst]: Sure, I'll send PRs.

File: index.rst (hunk 3 above)
Comment (begriffs) [reply index.rst]: The change is not actually related to this PR per se, I was just throwing it in as a little fix.

I noticed that currently on the master branch the forking document is present but it's not listed in the menu. It was removed from the menu when we added the new front page UI (and is linked from the Cloud "portal" page in docs). I don't remember us talking about removing forking from the main menu, so it seemed like a mistake.
